plugin-tester/src/runner.js

```diff
@@ -9,18 +16,19 @@ const ignoredResponse = {
     }
 }
 
-module.exports.local = (path, port, configReader) => () => {
+module.exports.local = (pluginPath, port, configReader) => () => {
     try {
         const config = configReader()
         const request = {
             body: {
                 type: 'query',
                 callbackUrl: `http://localhost:${port}/callback`,
-                configuration: config
+                configuration: config,
+                pluginInstanceId: config.pluginInstanceId
             }
         }
-        decache(path)
-        require(path).plugin()(request, ignoredResponse)
+        decache(pluginPath)
+        require(pluginPath)(request, ignoredResponse)
```

Comment:
I may have missed it but this looks like its missing `.plugin(dependencies)`

Reply:
it is indeed. i had fixed this but forgot to push.